src/styles/blocks/style.scss

```diff
@@ -0,0 +1,38 @@
+.watch {
+  position: relative;
+  height: $watch-size;
+  width: $watch-size;
+  background-image: url("../image/watch.png");
+  background-position: center;
+  border-radius: 50%;
+
+  &__small-hand {
+    position: absolute;
+    left: $left-position;
+    top: 190px;
```

Comment:
What will be if we will change clock size?

Reply:
get it, thanks